src/plugins/navcurrent/navcurrent.js

```diff
@@ -74,14 +74,19 @@ var componentName = "wb-navcurr",
 						// Pre-process the breadcrumb links
 						localBreadcrumbLinksArray = [];
 						localBreadcrumbLinksUrlArray = [];
-						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "a" );
-						len = localBreadcrumbLinks.length;
-						for ( i = 0; i !== len; i += 1 ) {
-							link = localBreadcrumbLinks[ i ];
-							linkHref = link.getAttribute( "href" );
-							if ( linkHref.length !== 0 && linkHref.charAt( 0 ) !== "#" ) {
-								localBreadcrumbLinksArray.push( link );
-								localBreadcrumbLinksUrlArray.push( link.hostname + link.pathname.replace( /^([^\/])/, "/$1" ) );
+						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "li" );
+						if( localBreadcrumbLinks ) {
+							len = localBreadcrumbLinks.length;
+							if ( len && len > 0 ) {
+								link = localBreadcrumbLinks[ ( len - 1 ) ];
```

Comment:
@duboisp From my research this is the only way to get the last element of an array in JavaScript so this line cannot change.

Sources:
http://www.codeblocq.com/2016/05/Get-the-last-element-of-an-Array-in-JavaScript/
http://www.w3resource.com/javascript-exercises/javascript-array-exercise-4.php
http://stackoverflow.com/questions/9050345/selecting-last-element-in-javascript-array